.github/workflows/build-test-push-workflow.yml

```diff
@@ -238,6 +234,8 @@ jobs:
       - name: Run smoke test
         id: smoketest
         run: |
+          export SPLUNK_OPERATOR_IMAGE_NAME=${{ env.SPLUNK_OPERATOR_IMAGE_NAME }}:$GITHUB_SHA
+          export SPLUNK_OPERATOR_IMAGE=${{ env.SPLUNK_OPERATOR_IMAGE_NAME }}:$GITHUB_SHA
```

Comment:
We have 2 env variables that are set to the same thing?

Reply:
I will remove one of them. export SPLUNK_OPERATOR_IMAGE_NAME=${{ env.SPLUNK_OPERATOR_IMAGE_NAME }}:$GITHUB_SHA will be removed